scripts/sshzobelisk

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
We're going with a convention of having a one line description at the top of each script, so we don't end up clueless in the future